Source/WebCore/rendering/RenderLayerScrollableArea.cpp

```diff
@@ -1751,6 +1752,9 @@ void RenderLayerScrollableArea::clearScrollCorner()
 
 void RenderLayerScrollableArea::updateResizerStyle()
 {
+    if (!m_resizer && !m_layer.canResize())
```

Comment:
Do we really need both condition to early return or any of them (`&&` or `||` ) ?

Reply:
Based on my understanding of code, we need both since` m_resize` is for custom scrollbar resize while `canResize()` is for special case we have for 'iframe'. As for `&&` or `||`, I think `||` will cover more cases but I wanted to be failsafe and only kept similar to what Blink commit did. Happy to change and push new commit. I am doing local build with `||` in background to confirm it works and does not cause any issue.

Reply:
Local build across various website seems to work fine and does not seems have any broken resizing behavior while changing to `||` and it would cover both cases while if either of them are true, which makes more sense. So I pushed new commit with change. Let's see what happens in EWS. Thanks for review!

Reply:
Since `||` will cover more cases in individually. I would opt for it and it does not regress anything on EWS as well. Marking this as "RESOLVED".